SQL/0000-00-02-Permission.sql

```diff
@@ -105,8 +108,8 @@ INSERT INTO `permissions` VALUES
     (33,'genomic_data_manager','Genomic Files',(SELECT ID FROM modules WHERE Name='genomic_browser'),'Upload','2'),
     (34,'media_write','Candidate Media Files',(SELECT ID FROM modules WHERE Name='media'),'Edit/Upload/Hide','2'),
     (35,'media_read','Candidate Media Files',(SELECT ID FROM modules WHERE Name='media'),'View/Download','2'),
-    (36,'issue_tracker_own_issue', 'See/Edit/Comment and Close on Own Issues',(SELECT ID FROM modules WHERE Name='issue_tracker'),NULL, 2),
-    (37,'issue_tracker_all_issue', 'See/Edit/Comment on all Issues',(SELECT ID FROM modules WHERE Name='issue_tracker'),NULL, 2),
+    (36,'issue_tracker_own_issue', 'Issues - Own',(SELECT ID FROM modules WHERE Name='issue_tracker'),'View/Edit/Comment/Close', 2),
```

Comment:
@shonibare can you confirm that the OWN means own issues and not OWN site issues? I'm worried one of these might be incorrect?

Reply:
<img width="437" alt="Screenshot 2025-02-19 at 3 10 23 PM" src="https://github.com/user-attachments/assets/dce63ea5-1750-4a4a-a54e-99052027f1f5" />